### ISSUE TITLE
Missing reform modules

### ISSUE THREAD

**Issue by MattiSG** [openfisca/openfisca-web-api#10]
Tests of `web-api` fail because of missing modules.

Steps for reproduction:

```
$ git clone git@github.com:openfisca/openfisca-web-api.git
$ cd openfisca-web-api
$ sudo pip install -r requirements.txt
$ make test
ERROR: test suite for <module 'openfisca_web_api.tests.test_calculate' from '/Users/sgmap/Dev/webapitest/openfisca_web_api/tests/test_calculate.py'>
[…]
ValueError: {u'revenu_de_base_enfants': u'No module named openfisca_france_reform_revenu_de_base_enfants', u'revenu_de_base_cotisations': u'No module named openfisca_france_reform_revenu_de_base_cotisations'} for: OrderedDict([(u'inversion_revenus', <function build_reform at 0x1049ac2a8>), (u'revenu_de_base_cotisations', [u'openfisca_france_reform_revenu_de_base_cotisations', u'build_reform']), (u'revenu_de_base_enfants', [u'openfisca_france_reform_revenu_de_base_enfants', u'build_reform'])])
```

**Comment by MattiSG** [comment]
This should be fixed with #18.

**Comment by MattiSG** [comment]
Fixed by 9717b83775886894141e205720486eb600556bac.
